### DISCUSSION TITLE
[firebase_database] how to read

### DISCUSSION THREAD

**Discussion by TheHypnoo** [firebase/flutterfire#7632] [Q&A] [answered]
Hello, with the new update of firebase_database (version ^ 9.0.0) the values ​​are returned to you as an Object, but in this case if I try to make a Map <String, dynamic> as previously happened now it prevents me. What is the correct way to be able to read more than one field?

Thanks.

**Comment by deepak786** [comment, marked as answer]
cast the value to map
```
ref.onValue.map((event) {
    var value = (event.snapshot.value ?? {}) as Map;
});
```
